cnf-certification-test/certification/suite.go

```diff
@@ -202,3 +206,14 @@ func testContainerCertificationStatusByDigest(env *provider.TestEnvironment, val
 	}
 	testhelper.AddTestResultLog("Non-compliant", failedContainers, tnf.ClaimFilePrintf, ginkgo.Fail)
 }
+
+func testHelmVerion(env *provider.TestEnvironment) {
+	helmVersion := env.HelmVersion
```

Comment:
I think the scope of this test is not understood in this PR.

We need to check each installed helm chart, to see whether or not they are v2 or v3.  Not the installed version in our container (which it isn't installed so we would have to add that).

Reply:
I agree, seems like the goal of this change is to check the APIversion used by the charts discovered. We could use the following to get the APIVersion, then check it :
helmchartsReleases := env.HelmChartReleases
	for _, helm := range helmchartsReleases {
		aVersion:=helm.Chart.Metadata.APIVersion